lib/fathead/mdn_css/cover/keywords.txt

```diff
@@ -0,0 +1,1203 @@
+:at-level
+-atsc-
+attr()
+auto
+::backdrop
+backface-visibility
+background
+background-attachment
+background-clip
+background-color
+background-composite
+background-image
+background-inline-policy
+background-origin
+background-origin-x
+background-origin-y
+background-position
+background-position-x
+background-position-y
+background-repeat
+background-size
+baseline-shift
+::before
+behavior
+:below-level
+:blank
+bleed
+blend-mode
+block-progression
+bookmark-label
+bookmark-level
+bookmark-state
+bookmark-target
+border
+border-after
+border-after-color
+border-after-style
+border-after-width
+border-before
+border-before-color
+border-before-style
+border-before-width
+border-bottom
+border-bottom-color
+border-bottom-colors
+border-bottom-left-radius
+border-bottom-right-radius
+border-bottom-style
+border-bottom-width
+border-collapse
+border-color
+border-end
+border-end-color
+border-end-style
+border-end-width
+border-fit
+border-horizontal-spacing
+border-image
+border-image-outset
+border-image-repeat
+border-image-slice
+border-image-source
+border-image-width
+border-left
+border-left-color
+border-left-colors
+border-left-style
+border-left-width
+border-radius
+border-right
+border-right-color
+border-right-colors
+border-right-style
+border-right-width
+border-spacing
+border-start
+border-start-color
+border-start-style
+border-start-width
+border-style
+border-top
+border-top-color
+border-top-colors
+border-top-left-radius
+border-top-right-radius
+border-top-style
+border-top-width
+border-vertical-spacing
+border-width
+bottom
+box-decoration-break
+box-direction
+box-flex
+box-flex-group
+box-lines
+box-ordinal-group
+box-orient
+box-pack
+box-reflect
+box-shadow
+box-sizing
+break-after
+break-before
+break-inside
+:broken
+::browse
+calc()
+::canvas
+canvas()
+caption-side
+::cell-content
+ch
+chains
+@charset
+::check
+:checked
+::choices
+clear
+::clear
+clip
+clip-path
+clip-rule
+cm
+color
+<color>
+color-index
+::column()
+column-axis
+column-break-after
+column-break-before
+column-break-inside
+column-count
+column-fill
+column-gap
+column-progression
+column-rule
+column-rule-color
+column-rule-style
+column-rule-width
+columns
+column-span
+column-width
+content
+content-zoom-chaining
+content-zooming
+content-zoom-limit
+content-zoom-limit-max
+content-zoom-limit-min
+content-zoom-snap
+content-zoom-snap-points
+content-zoom-snap-type
+@counter
+counter()
+counter-increment
+counter-reset
+counters()
+@counter-style
+crop
+cross-fade()
+cubic-bezier()
+cue
+cue-after
+cue-before
+:current
+cursor
+dashboard-region
+default
+:default
+deg
+device-aspect-ratio
+device-cmyk()
+device-pixel-ratio
+device-width
+:dir()
+direction
+:disabled
+display
+@document
+dominant-baseline
+dpcm
+dpi
+dppx
+:drag-over
+drop-initial-after-adjust
+drop-initial-after-align
+drop-initial-before-adjust
+drop-initial-before-align
+drop-initial-size
+drop-initial-value
+element()
+em
+:empty
+empty-cells
+:enabled
+-epub-
```

Comment:
@hchienjo where does this come from? I don't believe we expect to cover this, so we can remove this and the others that are similar (e.g. `-atsc-`)